go-controller/pkg/ovn/zone_interconnect/chassis_handler.go

```diff
@@ -152,5 +153,10 @@ func (zch *ZoneChassisHandler) createOrUpdateNodeChassis(node *corev1.Node, isRe
 		Options:     map[string]string{"csum": "true"},
 	}
 
+	// set the geneve port if using something else than default
+	if config.Default.EncapPort != config.DefaultEncapPort {
```

Comment:
do we need a log here to confirm this step is done? better debugging i guess, but if this is notified elsewhere differently I am ok not doing this here.